pdr_backend/pdr_dashboard/pages/feeds.py

```diff
@@ -95,10 +68,9 @@ def get_metrics_row(self):
             children=[
                 get_metric(
                     label=key,
-                    value=value,
                     value_id=f"feeds_page_{key}_metric",
                 )
-                for key, value in self.app.data.feeds_metrics_data.items()
+                for key in ["Feeds", "Accuracy", "Volume", "Sales", "Revenue"]
```

Comment:
I think we can define these in constants. What do you think? We have col headers there, so we might as well have metrics names